ci/test/04_install.sh

```diff
@@ -6,6 +6,29 @@
 
 export LC_ALL=C.UTF-8
 
+if [ "$TRAVIS_OS_NAME" == "osx" ]; then
+  cd /usr/local/Homebrew || exit 1
+  git reset --hard origin/master
+  ${CI_RETRY_EXE} brew update
```

Comment:
@MarcoFalke `brew update` blew up on a previous run (can't find it anymore) and recommended these two lines to "fix" it.

Reply:
It might be worth rebasing the PR for a while to see if any other surprises show up.